microservices/netinfra_utils/workers/vpls_service_worker.py

```diff
@@ -241,8 +247,30 @@ def device_add_to_vpls_dryrun(task):
 
 def device_add_to_vpls(task, commit_type):
     vpls_config = read_remote_services(task)
+    service_to_add = Service.parse_from_task(task)
+
+    ifc_put_responses = []
```

Comment:
Just to clarify: This block of code basically makes sure to configure an interface of newly added device right ?

If so, are we not missing ifc_reset ? 
also, this seems like copy&pasted code, should we reuse ? because it should be the same what happens when we create a new VPLS instance